mix.exs

```diff
@@ -19,7 +19,7 @@ defmodule Postgrex.Mixfile do
 
   # Configuration for the OTP application
   def application do
-    []
+    [applications: [:logger]]
```

Comment:
Nah, it is fine. If you want you can also start it in your test_helper but it is fine.